Mobile version Skeleton

FabricioLatl opened the pull request:
Pull request to merge with main, First pull request to merge index HTML file, main CSS file and Readme .

Features added: skeleton for mobile version of portfolio project

Commit:
First commit

First setup of readme and html + css files.

Commit:
HTML Navbar Setup

Navbar setup, SVG imports

Commit:
Finished CSS Responsiveness

Finished CSS responsiveness
Finished Navbar
Finished color tweaks

Commit:
Change indentation

Indentation fixes in order to comply with linter

Commit:
Update to stylistic code

Updates to code in order to comply with linter.

Commit:
Stylistic updates

Updates to HTML and CSS in order to comply with linter

isaacpitwa reviewed (changes requested):
## Invalid Request 🛑

Hi @FabricioLatl 👋🏽👋🏽,

This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you implement the following change

**Friendly reminder:** You should use Github Flow as stated in the requirements of this project. Please take a step back and read a lesson [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). 
- Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).
- This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@isaacpitwa** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FabricioLatl commented:
Hi  @isaacpitwa!

I do have a couple questions, what exactly do i not have about github flow on this repo? the style con the code is correct, i have a readme file , i do not think i need a gitignore file, the commit messages are clean and only the necessary files are in the repo

Please let me know so i can learn :3 

thank you

Fabricio

isaacpitwa commented:
hi @FabricioLatl ,

Unfortunately, your Pull request has a less meaningful title (not specific of which feature has been developed) plus also the submitted pull request doesn't have a description of the changes made.

ShahierNashaat commented on index.html:
```diff
@@ -0,0 +1,59 @@
+           
+        </div>
+
+    </header>
```
- [x] Kindly be noticed that the `header` section should contain the following.
- The logo is on the left and has a link (a tag).
- The hamburger button is on the right and has a `nav` tag.
- [x] Also notice that the header has the proper background image.

ShahierNashaat commented on index.html:
```diff
@@ -0,0 +1,59 @@
+        <div class="art">
+            <img alt="art" src="images/art.svg">
+        </div>
+    </div>
```
- [x] Kindly be noticed that the `div` that has the class `body-container` should be a `section` tag instead of `div` tag.
- [x] Also you should add the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
- [x] Also notice that the decorations are always background images, not `img` tags.

ShahierNashaat reviewed (changes requested):
Hi @FabricioLatl , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly be noticed that you should implement the mobile size design what you have implemented is a desktop size design.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Added Media querys to Styles.css

Added media querys for mobile responsivness

Commit:
Update Styles.css

Linter updates

FabricioLatl commented:
It now works with media querys in order for mobile design! It should look good on a >768px screen

mukhammadrizooff reviewed (changes requested):
Hi @FabricioLatl , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good Implementations ✔️ 
- No linter errors ✔️
- Good design ✔️  

## Required Changes ♻️
- [ ] [REQUIRED] Please make the changes [suggested by the previous Code Reviewer](https://github.com/FabricioLatl/Mobile-Portfolio/pull/1#pullrequestreview-1025026926)! Becouse you only change `media query` size. Please make changes!

![image](https://user-images.githubusercontent.com/63915024/176725533-2fc06cf7-bbc0-4648-be06-a131c5bc44e1.png)

![image](https://user-images.githubusercontent.com/63915024/176725563-4ab819dd-df59-4ef4-85c2-bb60e2a3f712.png)



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FabricioLatl commented:
hi! @mukhammadrizooff 
I do not understand :c 
I thought implementing mobile design meant utilizing media queries in order to make the site work responsively on a small screen. 
that's why I made the change and it looks pretty good now on a small screen

please help me by letting me know what I'm missing

mukhammadrizooff commented:
> * make the changes [suggested by the previous Code Reviewer](https://github.com/FabricioLatl/Mobile-Portfolio/pull/1#pullrequestreview-1025026926)!

Hello @FabricioLatl!
Please check these comments: [Comment-1](https://github.com/FabricioLatl/Mobile-Portfolio/pull/1#discussion_r911131807), [Comment-2](https://github.com/FabricioLatl/Mobile-Portfolio/pull/1#discussion_r911135222) by previous reviewer. Please check again these requirements!!!

Commit:
Updated Header

Header changes, background image on header. kiwidev logo now has an href

Commit:
Update to styles

Styles.css changed, img now background image, svg changed opacity, media query for grid now deleted.

Commit:
update stylescss

Linter issues.

BertrandConxy commented on index.html:
```diff
@@ -0,0 +1,62 @@
+
+    <section class="body-container">
+   
+
+      <ul class="intro">
+        <li>
+          <h1 class="name">
+              Hello! I'm Fabricio La Torre.
+          </h1>
+          <h3>
+              I'm a Web Developer 
+          </h3>
+          <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nobis doloremque ea sequi ullam dolor animi, cumque ab voluptate iusto sit.
+          </p>
+        </li>
+    
+      </ul>
```
- [ ] It is important to acknowledge what you have done so far. Great job 👏. In this section, it looks like you misunderstood what the reviewer was trying to tell you. If you try to view your work before adding these changes 👇 
![image](https://user-images.githubusercontent.com/90222110/176766355-181872e4-accc-481f-ad66-feff330dc411.png)
This was correct and you didn't have to change it. What he wanted to say was to add the social media icons and also group them in list. Look below the text paragraph in Figma design. Picture below 👇 
![image](https://user-images.githubusercontent.com/90222110/176767360-f377ea96-8fe6-4a62-97fd-14ce3b427efd.png)
![image](https://user-images.githubusercontent.com/90222110/176768736-a5292f44-c57b-4504-9f09-03ae89560006.png)

These are the ones you have to add using list tags and they should be links. Let me give you the format which you can refer to while you can build them.
      `<ul>
            <li><a href=' URL to your profile'><img href='PATH TO THE IMAGE' /></a></li>
            <li><a href=' URL to your profile'><img href='PATH TO THE IMAGE' /></a></li>
      </ul>`
 Then after, you will style them correctly. Note that you will use `Flexbox` to layout them 😃 💡

BertrandConxy reviewed (changes requested):
Hi @FabricioLatl, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes and I really appreciate that you tried to interact with them in order to give you more information. You did it well so far, but there are still some which aren't addressed yet. But you are so close 🏃‍♂️ 

 ## Highlights 💪 
- Good communication with reviewers
- All linters are passing
- Good README
- project is working on my PC

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] I would like you to watch this [Loom video](https://www.loom.com/share/657db95c7ec1493b8f13691119da8c2a) I recorded where I was comparing your design with the Figma design given and giving you some advice to improve your design. Please watch it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Added Socials buttons

Added socials buttons to page: Github Twitter and Linkedin

Commit:
Update index.hrml

Linter errors

Commit:
Update index.html

Lucas-Masaba commented on index.html:
```diff
@@ -0,0 +1,69 @@
+                
+            </h1>
+            <nav class="main-nav">
+               
+                <ul class="main-nav-list">
+                  
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="#"><img alt="maillogo" src="images/mail.svg"></a>
+                    </li>
+                </ul>
```
- [ ] Please remove the `ul` tag with everything in it and replace it with a hamburger button as seen in the mobile version of the Figma template. Right now we are working on the mobile version and this is only needed in the desktop version of the portfolio 👍

Lucas-Masaba reviewed (changes requested):
# Hello @FabricioLatl  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-there](https://media.giphy.com/media/ReyD4OdRiHeLDCRaEP/giphy.gif)

## Highlights 🍾 
- Linter checks passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Currently, your portfolio looks like this 
![image](https://user-images.githubusercontent.com/45460192/177588633-bdc3dca3-d6e8-4ca9-9bf0-5a02380db64d.png)
Make sure it looks similar to the Figma and eliminate the horizontal scroll 
![image](https://user-images.githubusercontent.com/45460192/177589096-a03f4724-eb19-4f0c-88a4-51b7ec3b6088.png)

Commit:
Added Hamburguer button

GzimAsani commented on index.html:
```diff
@@ -0,0 +1,73 @@
+    <title>Fabricio La Torre</title>
+</head>
+<body>
+    <header class="main-header">
```
- [ ] You still have work to do in order to match it with the given design
![image](https://user-images.githubusercontent.com/62705947/177800979-233f049f-dfe8-400d-bb9b-984bd9d38e15.png)

barakadanny replied:
I guess this may be a mistake, since I'm not the one working on the project but I'm still open to help if needed thank you 
@GzimAsani @FabricioLatl

GzimAsani reviewed (changes requested):
### Hello @FabricioLatl     :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * I know that you are working hard, you just have few more things to make :pray: 
 
 ## Required Changes 

- [ ] _Check the comments below :arrow_double_down:_



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Stylistic changes to design and color pallete

Commit:
Update styles.css

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,73 @@
+              <a href="#"><img alt="kiwilogo" src="images/kiwilogo.svg"></a>
+                
+            </h1>
+            <!--
+            <nav class="main-nav">
+               
+                <ul class="main-nav-list">
+                  
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="#"><img alt="maillogo" src="images/mail.svg"></a>
+                    </li>
+                </ul>
```
- [ ] Kindly remove any commented-out unwanted code from your code base as it makes it look cluttered and messy. Kindly refer to the [HTML and CSS best practices rule 12](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for reference.

Nemwel-Boniface reviewed (changes requested):
## Hello @FabricioLatl   :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Thank you for making some of the changes requested by previous reviewers. :clap: I would also like to commend your communication skills with reviewers when something is not clear. Keep this up :clap:
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️


## ♻️ Required Changes ♻️

- [ ] Kindly note that You are required to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). As requested by previous reviewers, kindly make sure that your portfolio section landing page matches the chosen Figma design.

**This is how your project currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/177852055-cf82f34f-feba-4e1b-b278-5fd5b43da66b.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/177852201-822eb6d0-52c1-45ca-817d-dd5f50ec64c8.png)

You are really close to making this milestone approved. Keep pushing a little longer. You got this :muscle: 

_Check the comments under the review._

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Final changes on font and navbar

Commit:
Changes to color on text and SVG

HAMMAS-SALEEM commented on index.html:
```diff
@@ -0,0 +1,73 @@
+              <a href="#">La Torre</a>
+                
+            </h1>
+            <!--
+            <nav class="main-nav">
+               
+                <ul class="main-nav-list">
+                  
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="#"><img alt="maillogo" src="images/mail.svg"></a>
+                    </li>
+                </ul>
```
**[Mandatory Change]**
- [ ] Please remove this commented code from your HTML. :+1: Because commented code makes your code not look clean :cry:

HAMMAS-SALEEM commented on index.html:
```diff
@@ -0,0 +1,73 @@
+      
+        
+      </ul>
+      <ul class="socials">
+        <li><a href='https://github.com/FabricioLatl/'><img alt="github" src='images/github.svg' /></a></li>
+        <li><a href='https://www.linkedin.com/in/fabriciolatl/'><img alt="linkedin" src='images/linkedin.svg' /></a></li> 
+        <li><a href='https://twitter.com/Wafer_lt'><img alt="twitter" src='images/twitter.svg' /></a></li>
+      </ul>
```
![all-icons](https://user-images.githubusercontent.com/93039359/177863836-f3e66ba1-3519-4ff8-9e85-accd0b4b6665.png)

**[Optional]**

- [ ] Please add all the icons as shown in the Figma template :+1: because your portfolio should contain all your social media links. :+1: by which recruiters will contact you :eyes:

HAMMAS-SALEEM commented on index.html:
```diff
@@ -0,0 +1,73 @@
+          <h3>
+              I'm a Web Developer 
+          </h3>
+          <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nobis doloremque ea sequi ullam dolor animi, cumque ab voluptate iusto sit.
+          </p>
```
![paragraph](https://user-images.githubusercontent.com/93039359/177864182-dacf8d4e-6f75-4433-913b-2b6388f9fc3e.png)

**[Mandatory Change]**

- [ ] Instead of Lorem ipsum please add the paragraph shown in the Figma template :+1: Because your portfolio is showing you so you should add this paragraph to say what you can do :muscle:

HAMMAS-SALEEM commented on index.html:
```diff
@@ -0,0 +1,73 @@
+<body>
+    <header class="main-header">
+        <div class="container">
+            <h1 class="mh-logo">
+              <a href="#">La Torre</a>
+                
+            </h1>
```
**[Optional]**

- [ ] You don't need to put your `a` tag inside `h1`. Because the `h1` tag is for page main heading and your name is not your page heading :thinking:

HAMMAS-SALEEM reviewed (changes requested):
# Required Changes :recycle:
![almost-there](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQp9z2Rq_bAJ42eRNxObBr1nKDUbZjLw2QqkQ&usqp=CAU)

### Hey @FabricioLatl :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow


Good job so far!
You have made a lot of effort in making changes requested by the previous reviewer but there are some issues that you still need to work on to go to the next project but you are almost there! :+1:

# Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Updated Index.html

comments deleted, as well as lorem ipsum replaced.
SVG file fixed

vechicin commented on styles/styles.css:
```diff
@@ -0,0 +1,138 @@
+  text-align: center;
+}
+
+.main-header {
+  background: #3e3c3c;
+  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.1);
+}
```
Your toolbar looks very dissimilar from that of the template design. Please review the Figma design and fix it.

vechicin commented on styles/styles.css:
```diff
@@ -0,0 +1,138 @@
+  display: inline;
+}
+
+.body-container {
+  display: grid;
+  position: absolute;
+  width: 100%;
+  margin-top: 15%;
+}
```
This section is very dissimilar from the Figma template design. Among the things you need to fix (but not exclusively): background image, social media icons, margins/paddings, and fonts. Please review the Figma template design and make sure your design is as similar as possible.

vechicin reviewed (changes requested):
# Hi @FabricioLatl ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
background changes

change to background image, font changes

Commit:
linter updates

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @FabricioLatl  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
